implants/lib/eldritch/src/sys/exec_impl.rs

```diff
@@ -1,132 +1,176 @@
+                    exit(0)
                 }
-            }
+
+                ForkResult::Child => {
+                    let _res = Command::new(path)
+                        .args(args)
+                        .output()
+                        .expect("failed to execute process");
+                    exit(0)
+                }
+            },
         }
     }
 }
 
 #[cfg(test)]
 mod tests {
-    use std::{path::Path, time, thread, fs};
+    use std::{fs, path::Path, thread, time};
 
     use tempfile::NamedTempFile;
 
     use super::*;
     #[test]
-    fn test_sys_exec_current_user() -> anyhow::Result<()>{
-        if cfg!(target_os = "linux") || 
-        cfg!(target_os = "ios") || 
-        cfg!(target_os = "android") || 
-        cfg!(target_os = "freebsd") || 
-        cfg!(target_os = "openbsd") ||
-        cfg!(target_os = "netbsd") {
-            let res = handle_exec(String::from("/bin/sh"),vec![String::from("-c"), String::from("id -u")], Some(false))?.stdout;
-            let mut bool_res = false; 
+    fn test_sys_exec_current_user() -> anyhow::Result<()> {
+        if cfg!(target_os = "linux")
+            || cfg!(target_os = "ios")
+            || cfg!(target_os = "android")
+            || cfg!(target_os = "freebsd")
+            || cfg!(target_os = "openbsd")
+            || cfg!(target_os = "netbsd")
+        {
+            let res = handle_exec(
+                String::from("/bin/sh"),
+                vec![String::from("-c"), String::from("id -u")],
+                Some(false),
+            )?
+            .stdout;
+            let mut bool_res = false;
             if res == "1001\n" || res == "0\n" {
                 bool_res = true;
             }
-            assert_eq!(bool_res, true);
-        }
-        else if cfg!(target_os = "macos") {
-            let res = handle_exec(String::from("/bin/echo"),vec![String::from("hello")], Some(false))?.stdout;
+            assert!(bool_res);
+        } else if cfg!(target_os = "macos") {
+            let res = handle_exec(
+                String::from("/bin/echo"),
+                vec![String::from("hello")],
+                Some(false),
+            )?
+            .stdout;
             assert_eq!(res, "hello\n");
-        }
-        else if cfg!(target_os = "windows") {
-            let res = handle_exec(String::from("C:\\Windows\\System32\\cmd.exe"), vec![String::from("/c"), String::from("whoami")], Some(false))?.stdout;
+        } else if cfg!(target_os = "windows") {
+            let res = handle_exec(
+                String::from("C:\\Windows\\System32\\cmd.exe"),
+                vec![String::from("/c"), String::from("whoami")],
+                Some(false),
+            )?
+            .stdout;
             let mut bool_res = false;
-            if res.contains("runneradmin") || res.contains("Administrator") || res.contains("user") {
+            if res.contains("runneradmin") || res.contains("Administrator") || res.contains("user")
+            {
```

Comment:
Let's keep with the convention of { on the same lien as the if.
If the line is too long we can break the conditions into a multiline block.

Reply:
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#alwayssameline-default